pkg/controller/provider/web/ova/workload.go

```diff
@@ -60,6 +65,10 @@ func (h WorkloadHandler) Get(ctx *gin.Context) {
 	}
 	r := Workload{}
 	r.With(m)
+	err = r.Expand(db)
+	if err != nil {
+		return
```

Comment:
shouldn't we to set the status to http.StatusInternalServerError?

Reply:
isn't that covered by the defer inside this func? since we using the same `err` variable in the same scope I think it should be catched there.

Reply:
ah ok, didn't notice that part when reviewing the changes - so that's fine